### DISCUSSION TITLE
N64 fastcrc calculating incorrectly

### DISCUSSION THREAD

**Discussion by Cephiros** [sanni/cartreader#610] [Need help]
Subject says it all. Fastcrc is not correct. With fastcrc off the slow method works correctly. With it on dumps dont validate, however manual calculating after the fact and searching that within n64.txt usually returns a valid result. This is true for sure with hardware5 and version 10.2, and the prior version, 9.3 was it? I havent tried prior versions.

**Comment by sanni** [comment]
Try another SD card. I'm not sure what exactly makes an SD card good for use with the OSCR but some work with fastcrc and some don't. If you by any chance have an old card between 4 and 32GB, freshly formatted to FAT32 you can try that.

**Comment by PsyK0p4T** [comment]
After multiple reformat operations on mines, some SD ended to not work perfectly anymore with OSCR (incorrect rom CRCs etc). Couldn't explain why. Finally I used https://www.sdcard.org/downloads/formatter/, fat32, and all settings set on default. My SD worked great again, for all kind of operations (dumping, copying, fastcrc etc.)
Feel free to try, nothing to loose :/
